packages/core-mobile/app/navigation/TopNavigationHeader.tsx

```diff
@@ -111,7 +113,7 @@ const TopNavigationHeader: FC<Props> = ({
   )
 
   return (
-    <View>
+    <View style={{ paddingTop: insets.top }}>
```

Comment:
<img width="1465" alt="Screenshot 2024-09-03 at 11 07 00 PM" src="https://github.com/user-attachments/assets/44186a11-2f84-4757-bdb8-b124d2944c67">

I did some more testing on Android and it looks like the bottom tab + top navigation header + floating action button still need adjustments. In the pic above, I took a few Android screenshots of the current Core Mobile as well as some popular apps for your references. Basically, for those 3 components, we just want our app to always match the others as best as we can. Let me know if you have any questions.

Reply:
Padding wasn't dynamic for the button before so I simply adjusted it, but I'll make it dynamic and that should look good at all times.